common/containers/Tabs/Swap/components/currentRates.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import translate from 'translations';
+import PropTypes from 'prop-types';
+
+
+export default class CurrentRates extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
Hmm.. any specific reason? Just hate constructors? This is a pretty common pattern.